### DISCUSSION TITLE
use each block as separate in same module

### DISCUSSION THREAD

**Discussion by devimako** [area17/twill#2221] [Q&A]
## Description
how to separate the two blocks in the same module because i am getting all the block if i use in the same module
i need to use this blocks as separate for each section in page
### Actual result

i need to use the different block as a separate in each section in same module form 
<img width="1423" alt="Screenshot 2023-02-13 at 12 20 39 PM" src="https://user-images.githubusercontent.com/99637767/218447143-382f411c-82ab-46a7-b6c9-124f47079419.png">
<img width="691" alt="Screenshot 2023-02-13 at 12 23 10 PM" src="https://user-images.githubusercontent.com/99637767/218447170-71a437ed-657c-4140-ae06-391736d09f44.png">

### Versions

 Twill version:  2.12
 Laravel version: 9
 PHP version: 8.2
Database engine: pgsql

**Comment by ifox** [comment]
Hi @devimako, you can use multiple block editor fields on the same form, with different allowed blocks. Are you sure you are seeing 2 blocks in the drop-down with the code you shared where you specified a single block type?
